conda/pytorch-cuda/meta.yaml

```diff
@@ -11,15 +11,7 @@
 # deployment:
 # https://conda.anaconda.org/pytorch/noarch/
 # https://conda.anaconda.org/pytorch/noarch/repodata.json
-{% set build = 5 %}
-{% set cuda_constraints=">=11.7,<11.8" %}
-{% set libcufft_constraints=">=10.7.2.50,<10.9.0.58" %}
-{% set libcublas_constraints=">=11.10.1.25,<11.11.3.6" %}
-{% set libcusolver_constraints=">=11.3.5.50,<11.4.1.48" %}
-{% set libcusparse_constraints=">=11.7.3.50,<11.7.5.86" %}
-{% set libnpp_constraints=">=11.7.3.21,<11.8.0.86" %}
-{% set libnvjpeg_constraints=">=11.7.2.34,<11.9.0.86" %}
-{% if version == '11.8' %}
+{% set build = 6 %}
```

Comment:
I believe I need to bump the build version, but please correct me if I'm wrong.

Reply:
Yes totally bump build version, however as Nikita mentioned please keep older versions of cuda here.